espanol/poseso/poseso.md

```diff
@@ -0,0 +1 @@
+Con los ojos muy abiertos y haciendo aspavientos con las manos, el Consejero gritaba ante la atónica mirada de la multitud...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.